sympy/printing/tests/test_pycode.py

```diff
@@ -161,3 +160,64 @@ def test_NumPyPrinter_print_seq():
     n = NumPyPrinter()
 
     assert n._print_seq(range(2)) == '(0, 1,)'
+
+
+def test_issue_16535_16536():
+    from sympy import lowergamma, uppergamma
+
+    a = symbols('a')
+    expr1 = lowergamma(a, x)
+    expr2 = uppergamma(a, x)
+
+    prntr = SciPyPrinter()
+    assert prntr.doprint(expr1) == 'scipy.special.gamma(a)*scipy.special.gammainc(a, x)'
+    assert prntr.doprint(expr2) == 'scipy.special.gamma(a)*scipy.special.gammaincc(a, x)'
+
+    prntr = NumPyPrinter()
+    assert prntr.doprint(expr1) == '  # Not supported in Python:\n  # lowergamma\nlowergamma(a, x)'
+    assert prntr.doprint(expr2) == '  # Not supported in Python:\n  # uppergamma\nuppergamma(a, x)'
```

Comment:
Should this be "Not supported in Numpy"?

Reply:
One may think so. Not sure why it prints as it does...

Reply:
Probably because of me inheriting the method, but it is not really clear where the text comes from. It is a generic function that takes the name from somewhere, so in C it says C instead.

Reply:
I think I've found it... There will be a new push soon.